SpatialGDK/Source/SpatialGDK/Public/SpatialView/ConnectionHandlers/SpatialOSConnectionHandler.h

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
This is all inlined functions? Looking for speed?

Reply:
This one should probably be split out. Was inline for just how it was created.